src/runtime_src/core/edge/drm/zocl/zocl_sysfs.c

```diff
@@ -51,7 +51,7 @@ static ssize_t kds_custat_show(struct device *dev,
 	struct drm_zocl_dev *zdev = dev_get_drvdata(dev);
 	ssize_t size = 0;
 	phys_addr_t paddr;
-	u32 usage;
+	u32 usage, status;
```

Comment:
Yes, it is a short cut to define two variables of the same type on a common line.  Unfortunately, this type of formatted does have a negative impact on future developers reading the code.   While more verbose, could you break the "usage" and "status" variables to have their own separate types.